package.json

```diff
@@ -33,5 +33,8 @@
   "workspaces": [
     "packages/*"
   ],
-  "packageManager": "yarn@3.5.0"
+  "packageManager": "yarn@3.5.0",
+  "dependencies": {
+    "zustand": "^4.3.6"
```

Comment:
zustand should be added to the project's (dapp?) package.json, not the root one.

Reply:
Correct! Solved